### PR TITLE
added python profiler when ProfileExecutions is enabled

### PR DESCRIPTION
**Pull request by MeirShpilraien**
- [x] tests
- [x] documentation

### CODE REVIEW

**Review by rafie — APPROVED** [review]

File: plugins/python/GearsBuilder.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import redisgears
 import copy
 import redisgears as rg
+import cProfile, pstats
```
Comment (chayim) [inline plugins/python/GearsBuilder.py:4]:
```
import cProfile
import pstats
```

**Comment by MeirShpilraien** [comment]
After @gkorland review, unified API to a single RG.PYPROFILE command with multiple subcommands.

File: plugins/python/GearsBuilder.py (hunk 1 above)
Comment (MeirShpilraien) [reply plugins/python/GearsBuilder.py]: Fixed

**Comment by MeirShpilraien** [comment]
API was changed according to @gkorland suggestion.

**Review by rafie — APPROVED** [review]

**Pull request merged**